### DISCUSSION TITLE
还是没有看懂怎么下载

### DISCUSSION THREAD

**Discussion by EliaS0625** [tgbot-collection/YYeTsBot#108] [Q&A] [answered]
有详细的教程吗 download只下载了源代码 不能变成客户端

**Comment by BennyThink** [comment, marked as answer]
https://github.com/tgbot-collection/YYeTsBot#%E5%B0%8F%E7%99%BD%E4%BD%BF%E7%94%A8

https://github.com/tgbot-collection/YYeTsBot/blob/master/DEVELOPMENT.md

**Reply by EliaS0625** [reply]
感谢
